Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -158,6 +158,14 @@ private object MapActivityToLiteDb(Activity activity, Context context, eExecuted
             }
             activity.Acts.ToList().ForEach(action => this.MapActionToLiteDb((GingerCore.Actions.Act)action, context, executedFrom));
             AR.ActionsColl.AddRange(liteDbActionList);
+            //counts
+            //AR.ChildExecutableItemsCount.Add(HTMLReportConfiguration.eExecutionStatisticsCountBy.Action.ToString(), activity.Acts.Count);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)